### PR TITLE
[HA] An empty replica breaks when asks for a full resync

### PR DESCRIPTION
**Pull request by carlos-rodrigues-8x8**
## What does this PR do?

This PR is meant to fix a bug on full synch request on  HA mode.

## Motivation

Contributing for evolving arcadedb.

## Related issues

https://github.com/ArcadeData/arcadedb/issues/2594

## Additional Notes

-

## Checklist

- [ ] I have run the build using `mvn clean package` command
- [ ] My unit tests cover both failure and success scenarios

### CODE REVIEW

File: package/src/main/scripts/server.sh [hunk 1]
```diff
@@ -100,4 +100,4 @@ exec "$JAVA" $JAVA_OPTS \
   $ARCADEDB_JMX \
   $ARCADEDB_SETTINGS \
   -cp "$ARCADEDB_HOME/lib/*" \
-  $ARGS $* "$@" com.arcadedb.server.ArcadeDBServer
+  $ARGS "$@" com.arcadedb.server.ArcadeDBServer
```
[the remark this answers is not in the document]
Comment (carlos-rodrigues-8x8) [reply package/src/main/scripts/server.sh]: Removed used variable

[the remark this answers is not in the document]
Comment (gramian) [reply package/src/main/scripts/server.sh]: We need to tightly review this change. I remember experimenting with that before and it led to problems.

[the remark this answers is not in the document]
Comment (carlos-rodrigues-8x8) [reply package/src/main/scripts/server.sh]: Which change @gramian? Removing $* or $ARGS?

[the remark this answers is not in the document]
Comment (gramian) [reply package/src/main/scripts/server.sh]: Removing `$*`

[the remark this answers is not in the document]
Comment (carlos-rodrigues-8x8) [reply package/src/main/scripts/server.sh]: Both `$*` `$@` are doing the same thing. Doesn't make sense to have both in the command. The slightly difference between both is `$*` expands to a single string containing all arguments separated by a space, while `$@`  is an array, but we are using it as string since it is between `"`. If you feel comfortable I can change to use `$*`.

[the remark this answers is not in the document]
Comment (gramian) [reply package/src/main/scripts/server.sh]: We can try, we just need to test if the various shells (including MacOS) support both, or if there are corner cases where it could break.

[the remark this answers is not in the document]
Comment (gramian) [reply package/src/main/scripts/server.sh]: Unfortunately I don't remember what the problem was, but I was using the same reasoning as you above.

File: package/src/main/scripts/server.sh [hunk 2]
```diff
@@ -100,4 +100,4 @@ exec "$JAVA" $JAVA_OPTS \
   $ARCADEDB_JMX \
   $ARCADEDB_SETTINGS \
   -cp "$ARCADEDB_HOME/lib/*" \
-  $ARGS $* "$@" com.arcadedb.server.ArcadeDBServer
+  "$@" com.arcadedb.server.ArcadeDBServer
```
Comment (lvca) [inline package/src/main/scripts/server.sh]: @robfrank this looks fine to me, not sure about this change in the script. WDYT?

**Review by lvca — COMMENTED** [review]
This is ok for me, not sure about the change in the server.sh

File: package/src/main/scripts/server.sh (hunk 2 above)
Comment (carlos-rodrigues-8x8) [reply package/src/main/scripts/server.sh]: As I explained before to @gramian, both `$*` and `$@` doing the same thing, we are just duplicating the parameters.

Comment (carlos-rodrigues-8x8) [reply package/src/main/scripts/server.sh]: Reverted! Should I open another ticket to handle it?

Comment (lvca) [reply package/src/main/scripts/server.sh]: Yes please, thanks!
